tests/lib/linter/source-code-fixer.js

```diff
@@ -421,6 +429,22 @@ describe("SourceCodeFixer", () => {
 
         });
 
+        describe("Nullish fixes", () => {
+            it("should not throw if fix is null", () => {
+                const result = SourceCodeFixer.applyFixes(TEST_CODE, [NULL_FIX]);
+
+                assert.isFalse(result.fixed);
+                assert.strictEqual(result.output, TEST_CODE);
```

Comment:
```suggestion
                assert.strictEqual(result.output, TEST_CODE);
                assert.strictEqual(result.messages.length, 1);
                assert.strictEqual(result.messages[0].message, "null");
```

To ensure the message is not removed.